### ISSUE TITLE
Query callback isn't called

### ISSUE THREAD

**Issue by lpmmo4231** [hgourvest/node-firebird#54]
Hello,
I have a problem with a query, its callback is never called.

    var firebird = require('node-firebird');
    firebird.attach(options, function (err, db) {
       if (err)
            throw err;
        
        console.log("Connected to database");

        var articleByEanQuery = 'SELECT * ' +
            'FROM table';
        
        db.query(articleByEanQuery, function (err, result) {
            if (err) {
                console.log("Error");
                callback(err);
            }
            console.log("Query finished");
            //console.log(result);                     
            db.detach();
        }, logerror);
        
        console.log("finished");        
        return;
    });

I have set a breakpoint on the first line of the query callback but it is never reached. The client connects to the database correctly. The programm runs without any warning or error.

**Comment by lpmmo4231** [comment]
I now know, where the error is. I am trying to connect to a firebird 2.0.x database. I have tried this exact same setup with the current version (2.5.3) and it worked as expected. 

Unfortunately in my productional environment I have to connect to a 2.0.x firebird database, is there a way to query this database version?

thanks alot
konni

**Comment by mariuz** [comment]
2.0 and 2.1 are End of Life products 

http://www.firebirdsql.org/en/discontinued-versions/


My guess is related to charset issues , that is why we recommend firebird 2.5.x

https://github.com/hgourvest/node-firebird/issues/29#issuecomment-26835885
